lib/src/build/polymer_smoke_generator.dart

```diff
@@ -223,11 +223,13 @@ class PolymerSmokeGenerator {
   ///    * invoke methods in event handlers.
   _processClass(ClassElement cls, Recorder recorder) {
     if (!_hasPolymerMixin(cls)) return;
+    if (cls.node is! ClassDeclaration) return;
```

Comment:
ah, ok, thanks. I just found the bug you filed, I see it will change back in 0.24.2.